orangecontrib/text/datasets/url.tab

```diff
@@ -0,0 +1,19 @@
+topic	url_case_study	url

```

Comment:
In the first row we have `whatsapp	URL1	URL2` and I do not see how URL1 is related to topic `whatsapp`. Am I missing something? I am wondering about, why do we have two URLs in the same row, if they are not related. Why not have it like this:
`tag1		URL1`
`whatsapp	URL2`


> the first row should contain attribute names right?
Yes, that's the header.